Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -2192,6 +2192,7 @@ void DocumentLoader::loadMainResource(ResourceRequest&& request)
         // The main navigation load will trigger the registration of the client.
         if (m_resultingClientId)
             scriptExecutionContextIdentifierToLoaderMap().remove(m_resultingClientId);
+            mainResourceLoadOptions.replacesClientIdentifier = m_resultingClientId.object();
```

Comment:
Bad indent or missing { }.
Also I am not sure this is the right way to get the resultingClientId, given a FrameLoader has several document loaders.
It might be best to get the id from the frame document instead if there is such a document.

Reply:
> Bad indent or missing { }.

Ah ok, because it is now two lines of code in the conditional block...

> It might be best to get the id from the frame document instead if there is such a document.

Ok, I will look further into this. Also, the spec refers to this being the id of the active document env settings object, so one could look up which one is active for the frame?

Reply:
I currently have the current code, which still passes the current tests. Will look into the tests next.

`// Set the replaced client Id, unless the current frame is about:blank.
        if (m_frame && m_frame->document() && frameLoader()->stateMachine().committedFirstRealDocumentLoad())
            mainResourceLoadOptions.replacesClientIdentifier = m_frame->document()->identifier().object();`